cmd/experimental/kueue-viz/INSTALL.md

```diff
@@ -0,0 +1,107 @@
+# Installation with helm
+
+KueueViz can be installed using helm using the following command and 
+by ensuring that `enableKueueViz` is set to `true`:
+
+```
+KUEUE_VERSION=v0.11.0
+helm upgrade --install kueue oci://registry.k8s.io/kueue/charts/kueue \
+  --version=$KUEUE_VERSION
+  --namespace kueue-system \
+  --set enableKueueViz=true \
+  --create-namespace
+```
+
+
+# Build and Run locally
+
+If you want to run KueueViz locally for development or debugging purposes, you need to go
+through the following steps.
+
+## Prerequisites
+You need a kubernetes cluster running kueue.
+If you don't have a running cluster, you can create one using kind and install kueue using helm.
+
+```
+kind create cluster
+kind get kubeconfig > kubeconfig
+export KUBECONFIG=$PWD/kubeconfig
+helm install kueue oci://us-central1-docker.pkg.dev/k8s-staging-images/charts/kueue \
+            --version="v0.9.1" --create-namespace --namespace=kueue-system
```

Comment:
```suggestion
            --version="v0.11.1" --create-namespace --namespace=kueue-system
```

I'm wondering if we can update this version automatically.

Reply:
Maybe we can add this to https://github.com/kubernetes-sigs/kueue/blob/d5adf68e206d0c422e8ea1145936ac4388aa64e7/Makefile#L284.

Reply:
Yeah, for sure, please update the release step

Reply:
updating that